store/modules/podcasts/podcasts.getters.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  getPodcastById: state => value => {
+    if (state.podcastsList != null) {
+      var podacastFind = state.podcastsList.find(
+        podcast => podcast.id === value
+      )
+      if (podacastFind != null) {
```

Comment:
I'm not sure that this check is necessary, if this problem occure, it's probably because the code is "bad" somewhere else. Maybe you can simplify this function with this : 
```
getPodcastById: state => podcastId => isNil(state.podcastsList) ? null : state.podcastsList.find(podcast => podcast.id === podcastId)
```

Reply:
isNil is a function you can import from lodash, it's check if a variable is either null or undefined